src/ragger/firmware/touch/positions.py

```diff
@@ -0,0 +1,498 @@
+            "{": Position(100, 415),
+            "}": Position(140, 415),
+            "#": Position(180, 415),
+            "%": Position(220, 415),
+            "^": Position(260, 415),
+            "*": Position(300, 415),
+            "+": Position(340, 415),
+            "=": Position(380, 415),
+            # second line
+            "_": Position(40, 470),
+            "\\": Position(80, 470),
+            "|": Position(120, 470),
+            "~": Position(160, 470),
+            "<": Position(200, 470),
+            ">": Position(240, 470),
+            "$": Position(280, 470),
+            "`": Position(320, 470),
+            "\"": Position(360, 470),
+            # third line
+            "more_specials": Position(50, 525),
+            ".": Position(120, 525),
+            ",": Position(160, 525),
+            "?": Position(200, 525),
+            "!": Position(240, 525),
+            "'": Position(280, 525),
+            "back": Position(380, 525),
+            # fourth and last line
+            "change_layout": Position(70, 580),
+            " ": Position(250, 580),
+        },
+        Firmware.FLEX: {
+            # first line
+            "[": Position(24, 350),
+            "]": Position(72, 350),
+            "{": Position(120, 350),
+            "}": Position(168, 350),
+            "#": Position(216, 350),
+            "%": Position(264, 350),
+            "^": Position(312, 350),
+            "*": Position(360, 350),
+            "+": Position(408, 350),
+            "=": Position(456, 350),
+            # second line
+            "_": Position(48, 420),
+            "\\": Position(96, 420),
+            "|": Position(144, 420),
+            "~": Position(192, 420),
+            ">": Position(240, 420),
+            "<": Position(288, 420),
+            "$": Position(336, 420),
+            "`": Position(384, 420),
+            "\"": Position(432, 420),
+            # third line
+            "more_specials": Position(60, 490),
+            ".": Position(144, 490),
+            ",": Position(192, 490),
+            "?": Position(240, 490),
+            "!": Position(288, 490),
+            "'": Position(336, 490),
+            "back": Position(420, 490),
+            # fourth and last line
+            "change_layout": Position(72, 560),
+            " ": Position(312, 560),
+        }
+    },
+    "TappableCenter": {
+        Firmware.STAX: STAX_CENTER,
+        Firmware.FLEX: FLEX_CENTER,
+    },
+    "KeyboardConfirmationButton": {
+        Firmware.STAX: STAX_CENTER,
+        Firmware.FLEX: FLEX_BUTTON_UPPER_CENTER_MIDDLE,
+    },
+    "RightHeader": {
+        Firmware.STAX: STAX_BUTTON_UPPER_RIGHT,
+        Firmware.FLEX: FLEX_BUTTON_UPPER_RIGHT
+    },
+    "LeftHeader": {
+        Firmware.STAX: STAX_BUTTON_UPPER_LEFT,
+        Firmware.FLEX: FLEX_BUTTON_UPPER_LEFT
+    },
+    "CenteredFooter": {
+        Firmware.STAX: STAX_BUTTON_LOWER_MIDDLE,
+        Firmware.FLEX: FLEX_BUTTON_LOWER_MIDDLE
```

Comment:
This should be `FLEX_BUTTON_LOWER_LEFT`

Reply:
How so? It would no longer be a `CenteredFooter` if it's left. :thinking:

Reply:
Ah so all of these layouts are moved from centered (Stax) to left (Flex)?
Indeed we need to wrap this.

Reply:
In Flex it is now split indeed : 
![image](https://github.com/LedgerHQ/ragger/assets/105934250/5ccc268c-e00a-476b-a598-b8628f54f675)